tendrl/gluster_integration/flows/sync_aggregated_cluster_details/__init__.py

```diff
@@ -0,0 +1,21 @@
+from tendrl.commons.event import Event
+from tendrl.commons import flows
+from tendrl.commons.message import Message
+
+
+class SyncAggregatedClusterDetails(flows.BaseFlow):
+    def run(self):
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Starting flow SyncAggregatedClusterDetails"
```

Comment:
This msg is already logged by the commons job processor , you can remove from here